MetaMask/metamask-extension
# Allow user to speed up transaction with any nonce if enabled

Fixes: #6553

Explanation:  

 - adds Speed Up Any Transaction Flag into advanced settings
 - the flag is disabled by default
 - if the flag is enabled the logic in _useShouldShowSpeedUp_ hook continues without regard to _isEarliestNonce_ evaluation result
- includes only English description and title messages
 
Manual testing steps: 
  - enable Speed up any transaction in Advanced settings
  - submit a transaction with low gas price
  - submit another transaction with a low gas price
  - tx 1 speed-up button should appear 
  - tx 2 speed-up button should appear too
  - speed up tx 1
  - speed up tx 2
